articles/mdm-migration.md

```diff
@@ -76,12 +204,12 @@ Configuring the end user migration workflow requires a few additional steps.
 
 ##### End user experience
 
-1. The end user will receive a "Device Enrollment: &lt;organization&gt; can automatically configure your Mac." system notification within the macOS Notifications Center. 
-   
-2. After the end user clicks on the system notification, macOS will open the **System Setting > Profiles** and ask the user to "Allow Device Enrollment: &lt;organization&gt; can automatically configure your Mac based on settings provided by your System Administrator."
-  
+1. The end user will receive a "Device Enrollment: &lt;organization&gt; can automatically configure your Mac." system notification within the macOS Notifications Center.
```

Comment:
```suggestion
1. The end user will receive a "Device Enrollment: &lt;organization&gt; can automatically configure your Mac." system notification within the macOS Notifications Center.
```

Is this still the case? I though the end user sees a full screen dialog? (not a system notification in Notifications Center)